rpc/amqp-rpc.go

```diff
@@ -562,12 +562,14 @@ func NewAmqpRPCClient(clientQueuePrefix, serverQueue string, c cmd.Config, stats
 					responseChan, present := rpc.pending[corrID]
 					rpc.mu.RUnlock()
 
-					rpc.log.Debug(fmt.Sprintf(" [c<][%s] response %s(%s) [%s]", clientQueue, msg.Type, core.B64enc(msg.Body), corrID))
 					if !present {
-						// AUDIT[ Misrouted Messages ] f523f21f-12d2-4c31-b2eb-ee4b7d96d60e
-						rpc.log.Audit(fmt.Sprintf(" [c<][%s] Misrouted message: %s - %s - %s", clientQueue, msg.Type, core.B64enc(msg.Body), msg.CorrelationId))
+						// occurs when a request is timed out and the arrives
+						// afterwards
+						stats.Inc(clientQueuePrefix+".RPC.AfterTimeoutResponseArrivals", 1, 1.0)
```

Comment:
Just to be pedantic, to fit with the rest of the RPC stats could you format this like `"RPC.AfterTimeoutResponseArrivals."+clientQueuePrefix`.